### ISSUE TITLE
running in to the archlinux-docker

### ISSUE THREAD

**Issue by HuanzuoSP** [codehz/mcpeserver#63]
**Describe the bug**
docker is can not used to "sudo"&"systemctl"&"dbus"

**To Reproduce**
mcpeserver run in to the archlinux-docker for pteroactyl!

about pteroactyl:Pterodactyl is the open-source game server management panel built with PHP7, Nodejs, and Go. 

**Expected behavior**
mcpeserver be restricted of docker,but it can to run mcpeserver v0.6.4 of the "./mcpeserver run"

**Additional context**
How to used the "./mcpeserver run" for v0.7.6 as same as v0.6.4? Because the docker can not to use "sudo"&"systemctl"&"dbus"

**Comment by codehz** [comment]
@HuanzuoSP No... but you can try to mount the system bus /run/dbus/system_bus_socket
